schemas/context/media-timed-asset-reference.schema.json

```diff
@@ -21,11 +21,39 @@
           "format": "uri-reference",
           "description": "Identifier of the content, which can be used to tie back to other industry or CMS IDs."
         },
+        "dc:creator": {
```

Comment:
Shouldn't these be references to the master DC grammar and not copied here?

Reply:
Changed this with a reference to DC grammar.